src/components/ClientQuote/ClientQuote.tsx

```diff
@@ -1,6 +1,6 @@
 import styles from "./ClientQuote.module.scss";
 import finishLine from "../../assets/images/finishLine.jpg";
-import Image from "next/image";
+import Image from "next/legacy/image";
```

Comment:
This is because I used some props for Next image that are no longer recommended by Next. So it was showing a message in the console to use the legacy code

Reply:
Its just to get rid of the message in the console